LICENSE

```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2015 Stellar Development Foundation
```

Comment:
@tomquisel Are you okay adding this copyright statement given Scott's first commit (while this repo was still under his `nullstyle/go-xdr` repository was in 2015?

Note that the license if the ISC license, not the Apache 2 license we normally use, but I don't think it's worth attaching a different license just for our changes.

Could you 👍 this PR if you approve of making this change to the LICENSE and the use of the ISC license for our current and future contributions.